src/verify_cert.rs

```diff
@@ -90,36 +91,40 @@ fn build_chain_inner(
 
     // TODO: revocation.
```

Comment:
extremely unrelated to this PR: should we drop this comment now?

Reply:
I'll get this in a small follow-up